client/src/nonblocking/rpc_client.rs

```diff
@@ -4507,6 +4508,42 @@ impl RpcClient {
         parse_keyed_accounts(accounts, RpcRequest::GetProgramAccounts)
     }
 
+    /// Returns the stake minimum delegation, in lamports.
+    pub async fn get_stake_minimum_delegation(&self) -> ClientResult<u64> {
+        let instruction = solana_sdk::stake::instruction::get_minimum_delegation();
+        let payer = None;
+        let transaction = Transaction::new_with_payer(&[instruction], payer);
+        let response = self.simulate_transaction(&transaction).await?;
+        let return_data = response
+            .value
+            .return_data
+            .ok_or_else(|| ClientErrorKind::Custom("return data was empty".to_string()))?;
+        let (program_id, data) = (return_data.program_id, return_data.data);
```

Comment:
```suggestion
        let (program_id, (data, encoding)) = response
            .value
            .return_data
            .ok_or_else(|| ClientErrorKind::Custom("return data was empty".to_string()))?;
```

Reply:
Wdyt? Slightly less verbose, and avoids the need for tuple indexing on `data` below.

Reply:
Love it! Done.